test/basic/ReportOnlyTest.gmk

```diff
@@ -29,7 +29,7 @@ ifdef HEADLESS
 REPORT_EXPECT_PASS = 91
```

Comment:
```suggestion
REPORT_EXPECT_PASS = 92
```

At least on GHA there seem to be 92 tests that pass.

Reply:
I'll have to check your suggested change; my recollection is that all tests passed when run locally.

Reply:
The warnings are as expected when conditions are as found.

Reply:
> I'll have to check your suggested change; my recollection is that all tests passed when run locally.

The `else` branch (`HEADLESS != 1`) also has an incremented `REPORT_EXPECT_PASS` value: from `93`  to `94`.

Note that the merge with `upstream/master` still leads to the same error:
```
/bin/grep -s "Test results: passed: ${EXPECT_PASS}; failed: ${EXPECT_FAIL}; error: 88" /home/runner/work/jtreg/jtreg/build/test/ReportOnlyTest.jt.log  > /dev/null
Expect: Test results: passed: 91; failed: 40; error: 88
make: *** [../test/basic/ReportOnlyTest.gmk:45: /home/runner/work/jtreg/jtreg/build/test/ReportOnlyTest.ok] Error 1
```

Reply:
Fixed via https://github.com/openjdk/jtreg/pull/217/commits/5c990293d97e15d4b292052210f977338640df74